src/char.cr

```diff
@@ -122,6 +122,12 @@ struct Char
     self - other
   end
 
+  # Performs a `#step` in the direction of the _limit_. For instance:
```

Comment:
The external name of the parameter is `to`. Should we use that or the internal one? 🤔 

```suggestion
  # Performs a `#step` in the direction of _to_. For instance:
```

"limit" sounds better, and I suppose that's the entire reason for using an internal name. So it's probably fine as it is, just want to double check.

Reply:
Yeah, I also wonder about this and decided for _limit_ but without being entirely convinced